content/id/docs/tasks/run-application/force-delete-stateful-set-pod.md

```diff
@@ -0,0 +1,81 @@
+---
+reviewers:
+title: Hapus Paksa Pod StatefulSet
+content_type: task
+weight: 70
+---
+
+<!-- overview -->
+Laman ini menjelaskan bagaimana cara menghapus Pod yang menjadi bagian dari sebuah {{< glossary_tooltip text="stateful set" term_id="StatefulSet" >}}, dan menjelaskan pertimbangan yang harus diperhatikan saat melakukannya.
+
+
+## {{% heading "prerequisites" %}}
+
+
+* Ini merupakan tugas yang cukup rumit dan memiliki potensi melanggar beberapa properti yang melekat dari StatefulSet.
+* Sebelum melanjutkan, pastikan kamu paham dengan pertimbangan yang disebutkan di bawah ini.
+
+
+
+<!-- steps -->
+
+## Pertimbangan StatefulSet
+
+Pada operasi normal dari StatefulSet, **tidak pernah** ada kebutuhan untuk menghapus paksa sebuah Pod StatefulSet. [Pengontrol StatefulSet](/docs/concepts/workloads/controllers/statefulset/) bertanggung jawab terhadap pembuatan, _scaling_ dan penghapusan terhadap anggota dari StatefulSet. Pengontrol akan berusaha menjaga agar jumlah Pod yang ditentukan dari 0 hingga N-1 hidup dan siap sedia. StatefulSet memastikan bahwa, pada waktu kapanpun, akan ada minimal satu Pod dengan identitas yang telah ditetapkan berjalan pada klaster. Hal ini direferensikan sebagai semantik *at most one* yang disediakan StatefulSet.
```

Comment:
ID Docs is available.
Scaling can be translated into this https://kbbi.kemdikbud.go.id/entri/Penyekalaan
Nit! I prefer to keep “controller”
```suggestion
Pada operasi normal dari StatefulSet, **tidak pernah** ada kebutuhan untuk menghapus paksa sebuah Pod StatefulSet. [_Controller_ StatefulSet](/id/docs/concepts/workloads/controllers/statefulset/) bertanggung jawab terhadap pembuatan, penyekalaan, dan penghapusan terhadap anggota dari StatefulSet. _Controller_ akan berusaha menjaga agar jumlah Pod yang ditentukan dari 0 hingga N-1 hidup dan siap sedia. StatefulSet memastikan bahwa, pada waktu kapanpun, akan ada minimal satu Pod dengan identitas yang telah ditetapkan berjalan pada klaster. Hal ini direferensikan sebagai semantik *at most one* yang disediakan dalam StatefulSet.
```